init.zsh

```diff
@@ -2,14 +2,14 @@
 # Zim initializition
 #
 
-autoload -Uz is-at-least
-if ! is-at-least 5.2; then
-  print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
-  return 1
-fi
+# autoload -Uz is-at-least
+# if ! is-at-least 5.2; then
+#   print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
+#   return 1
+# fi
 
 # Define zim location
-ZIM="${ZDOTDIR:-${HOME}}/.zim"
+(( ! ${+ZIM_HOME} )) && ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
```

Comment:
Since we're already checking for an env var, maybe now is the time to also check the XDG config dir?

CC: @ericbn